test/component/list.cy.tsx

```diff
@@ -880,4 +880,255 @@ describe("List", () => {
       });
     });
   });
+
+  const LIST_GROUPS_OPENABLE: ListGroupContent[] = [
+    {
+      id: "recent-content",
+      title: "Recent Content",
+      subtitle: "Your latest activity",
+      items: [
+        {
+          id: "messages",
+          title: "Messages",
+          subtitle: "Check your inbox",
+          iconUrl: RiMailFill,
+          children:
+            "Stay connected with your contacts by checking and replying to your recent messages. Keep conversations organized and never miss important updates.",
+        },
+        {
+          id: "notifications",
+          title: "Notifications",
+          subtitle: "View Alerts",
+          iconUrl: RiNotification3Fill,
+          children:
+            "See what's new at a glance. Review recent alerts, mentions, and important reminders to stay on top of your activities.",
+        },
+        {
+          id: "calendar",
+          title: "Calendar",
+          subtitle: "Upcoming events",
+          iconUrl: RiCalendar2Fill,
+          children:
+            "View your scheduled events and upcoming meetings in one place. Manage your time effectively and plan your week with confidence.",
+        },
+      ],
+    },
+    {
+      id: "all-content",
+      title: "All Content",
+      subtitle: "With warning rightSideContent",
+      items: [
+        {
+          id: "home",
+          title: "Home",
+          subtitle: "Go to homepage",
+          iconUrl: RiHome2Fill,
+          children:
+            "Return to your main dashboard where you can quickly access all your essential tools, updates, and recent highlights in one glance.",
+        },
+        {
+          id: "profile",
+          title: "Profile",
+          subtitle: "View your profile",
+          iconUrl: RiUser3Fill,
+          children:
+            "Customize your personal information, update your avatar, and manage your account preferences to reflect your identity and style.",
+        },
+        {
+          id: "settings",
+          title: "Settings",
+          subtitle: "Adjust preferences",
+          iconUrl: RiSettings3Fill,
+          openable: true,
+          children:
+            "Modify your system preferences, manage privacy and notifications, and fine-tune your user experience to suit your workflow.",
+        },
+      ],
+    },
+  ];
+
+  context("openable ", () => {
+    context("with children", () => {
```

Comment:
please organize this into `when given` and add another context `without children`